gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/jupiter/core/v4/invoker/EndpointInvoker.java

```diff
@@ -111,11 +90,73 @@ private <T extends EndpointConnector> T resolveConnector(final ExecutionContext
             entrypointConnector.supportedModes()
         );
 
+        final String endpointTarget = ctx.getAttribute(ATTR_REQUEST_ENDPOINT);
+
+        if (endpointTarget != null) {
+            final String evaluatedTarget = ctx.getTemplateEngine().getValue(endpointTarget, String.class);
```

Comment:
I thought `getValue` was deprecated in favor of `eval` which return a Single 🤔

Reply:
Yes it is something I want to re-discuss for performance consideration. Let's find a slot to discuss it.